spec/install/post_bundle_message_spec.rb

```diff
@@ -81,6 +81,22 @@
         expect(out).to include(bundle_complete_message)
       end
     end
+
+    describe "with misspelled or non-existent gem name" do
```

Comment:
Oh, sorry... I am happy to have tests in the resolver, but I think we should have at least this one test that is the full integration style. Please don't remove this test!